### ISSUE TITLE
Response bodies not displayed in Waterfall html

### ISSUE THREAD

**Issue by leathej1** [sitespeedio/sitespeed.io#2551]
Dockerized Sitespeed 9.6.0

Config:
```
"chrome": {
    "includeResponseBodies": "html"
```

```
"firefox": {
    "includeResponseBodies": "html"
```

Response bodies are not listed in the waterfall html even with this enabled.

**Comment by soulgalore** [comment]
It looks like its some bug in how we use CDP, it seems to work for some URLs but not for all.

**Comment by leathej1** [comment]
Sorry - the Firefox part was an error on my part. Mine was actually set for 'true' instead of 'html.

**Comment by soulgalore** [comment]
Yep its broken in Chrome. To fix it we need to do a new Chrome HAR release and a small fix. The good thing is that we also can add JS/CSS, the bad is that it will take some time to fix it.
